### PR TITLE
[mock] delete user , add validation, check number for ID

### PR DESCRIPTION
**Pull request by giangdvhe163178**
image
image
image

### CODE REVIEW

**Comment by giangdvhe163178** [comment]
image
image

**Comment by giangdvhe163178** [comment]
image
image
image

File: src/main/java/com/example/capstone_project/config/GlobalExceptionHandler.java [hunk 1]
```diff
@@ -36,7 +48,7 @@ public ResponseEntity<List<ExceptionResponse>>  TodoException(Exception ex, WebR
                 exceptionResponseList.add(exception);
             }
 
-            return ResponseEntity.status(HttpStatus.BAD_REQUEST).body(exceptionResponseList);
+            return exceptionResponseList;
```
Comment (tuannqhe151337) [inline src/main/java/com/example/capstone_project/config/GlobalExceptionHandler.java:51]: Sửa lại thành return ResponseEntity.status(HttpStatus.BAD_REQUEST).body(exceptionResponseList);

Comment (tuannqhe151337) [reply src/main/java/com/example/capstone_project/config/GlobalExceptionHandler.java:51]: Sorry anh đọc nhầm

**Review by tuannqhe151337 — APPROVED** [review]
You have a captivating smile.

**Pull request merged**